doc/conf.py

```diff
@@ -3,14 +3,17 @@
 This configuration only affects single-package Sphinx documenation builds.
 """
 
-from documenteer.sphinxconfig.stackconf import build_package_configs
-import lsst.ts.ATWhiteLightSource
+from documenteer.conf.pipelinespkg import *  # type: ignore # noqa
```

Comment:
I would rather do noqa: F403, which ignores only import * problem; it's unlikely the line will pose any other problem, but just in case..